### DISCUSSION TITLE
Persistent config with Docker

### DISCUSSION THREAD

**Discussion by dillanspencer** [bluenviron/mediamtx#3509] [Questions]
### Question

Is there a way to have the config and paths persist across resets without using the mediamtx.yml file. 

I have a server container that dynamically adds paths and config using the mediamtx API but I want those changes to persist

**Comment by aler9** [comment]
this is currently unimplemented but is being tracked here: https://github.com/bluenviron/mediamtx/issues/1406

[a comment by an automated account is not shown]
